ops-bedrock/docker-compose-devnet.yml

```diff
@@ -83,8 +83,8 @@ services:
       --metrics.port=7300
       --pprof.enabled
       --rpc.enable-admin
-      --da-rpc=http://da:26659
-      --namespace-id=e8e5f679bf7116cb
+      --da-rpc=http://host.docker.internal:26659
```

Comment:
Is this required?

Reply:
no this was just for testing yesterday, didn't mean to still have this there. fixing